### PR TITLE
chore: setup semantic release

### PR DESCRIPTION
**Pull request by philsturgeon**
And other general prepping for public

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+3. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Environment:**
+ - Extension version: [e.g. 1.0.1]
```
Comment (P0lip) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Let's include VS Code version as well.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +0,0 @@
-# Change Log
-
-All notable changes to the "spectral" extension will be documented in this file.
```
Comment (P0lip) [inline CHANGELOG.md:3]: Any reason not to keep changelog?

Comment (philsturgeon) [reply CHANGELOG.md:3]: Semantic Release will populate release notes automatically making this a bit redundant. 

For Spectral and big projects we maintain develop branches and put in extra effort on CHANGELOGs but for little libs and things like this we want zero effort git-flow-release.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (philsturgeon) [reply .github/ISSUE_TEMPLATE/bug_report.md:27]: Done! Thank you

**Pull request merged**